local-cli/runIOS/runIOS.js

```diff
@@ -160,30 +170,44 @@ function buildProject(xcodeProject, udid, scheme, configuration = 'Debug', launc
       '-derivedDataPath', 'build',
     ];
     console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
+    let xcpretty, xcprettyProcess;
```

Comment:
Why do we need `xcpretty` process?

Reply:
It is to store a reference to the spawned child process to pipe the console output to. Needs to be declared at the same scope as the xcpretty variable to be accessible in subsequent blocks.

Reply:
Sorry, I meant "Why do we need `xcpretty` variable here?".

Reply:
(It looks redundant, we already have `*process`.)

Reply:
xcpretty is simply a boolean to show that it is available. Could possibly be renamed to be more specific if that is a point of confusion. 

One can only start the process if it is available. A couple other switches depend on it as well.

Reply:
Essentially I don't want a variable to possibly Have different types based on state. Variables are cheap, and I believe this makes it more clear.

Reply:
I am okay with both renaming or using `xcprettyProcess` as a flag.

Reply:
I think we should get rid of xcpretty and just use xcprettyProcess as the flag.